### PR TITLE
[Student][MBL-12909] Update empty states for rubric and files tabs in submission details

### PR DESCRIPTION
**Pull request by StephenBrough**
See ticket for the specs Jordan posted.

### CODE REVIEW

File: libs/pandautils/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -168,7 +168,7 @@
     <string name="noAssignments">No Assignments</string>
     <string name="noAssignmentsSubtext">It looks like assignments haven\'t been created in this space. Time to go out and explore.</string>
     <string name="noAssignmentsTeacher">It looks like no assignments have been published yet.</string>
-    <string name="noFiles">No Files</string>
+    <string name="noFiles">There are no files for this assignment</string>
```
Comment (CalvinKern) [inline libs/pandautils/src/main/res/values/strings.xml]: Seems like this string is used in other places that are not in an assignment. Probably need a new resource for the submission files string.

Comment (StephenBrough) [reply libs/pandautils/src/main/res/values/strings.xml]: Good catch, changing now!

Comment (Blackchai) [reply libs/pandautils/src/main/res/values/strings.xml]: This is also in pandautils! Move it to pandares please if you are going to change it :)

Comment (Blackchai) [reply libs/pandautils/src/main/res/values/strings.xml]: Nevermind :3

**Pull request merged**